server/models/schema.sql

```diff
@@ -801,6 +801,8 @@ CREATE TABLE `inventory_group` (
   `donation_account` mediumINT(8) UNSIGNED DEFAULT NULL,
   `expires` TINYINT(1) DEFAULT 1,
   `unique_item`  TINYINT(1) DEFAULT 0,
+  `tracking_consumption`  TINYINT(1) DEFAULT 1,
+  `tracking_expiration`  TINYINT(1) DEFAULT 1,
```

Comment:
Can you combine the `tracking_expiration` and `expires` column?  They are both the same thing.